.eslintignore

```diff
@@ -5,4 +5,5 @@ sw.js
 **/lib/plugin.js
 **/.nuxt
 **/coverage
-**/packages/chakra-ui-theme/src/*
\ No newline at end of file
+**/packages/chakra-ui-theme/src/*
```

Comment:
I think this `packages/chakra-ui-theme` package should also be linted also. We had previously excluded it for some reason, but I cannot recall.

Reply:
Okay, I'd remove it so it can be listed as well.